### ISSUE TITLE
about the autoregressive finetuning

### ISSUE THREAD

**Issue by zhonglidadi** [google-deepmind/graphcast#68]
I trained a very good one-step model, but when I expanded the model's autoregressive steps from 2 to 12, the loss for steps 2-12 hardly converged. For example, when training a model with step=2, the loss for the first step remained around 0.008, and the loss for the second step remained around 0.014. When I extended it to 10 steps, the loss for the first step still remained around 0.008, and the loss for the second step remained around 0.014, with no change. This leads to inaccurate predictions in the medium to long term. What could be the potential issue?

**Comment by alvarosg** [comment]
Unfortunately, there is not an obvious way for us to be able to help with this config.

I would recommend creating some fake data, and verifying that you can overfit to that fake data first.
